### ISSUE TITLE
Update reductive

### ISSUE THREAD

**Issue by danieldk** [finalfusion/finalfusion-rust#52]
The `reductive` dependency should be updated sometime soon:

* It fixes seeding of multiple clustering attempts, so that the `attempts` of `ff-quantize` actually has an effect.
* It updates `ndarray-linalg` to 0.11. The version that we currently use (0.10) forces static linking of OpenBLAS, which fails on systems that do not provide a static library (when the `system` feature is used). Without the `system` feature, the compiled OpenBLAS library sometimes misses `LAPACKE_` symbols. (Not sure yet what the circumstances are, possibly it does not fail if there is a static system library, because it links against that first.)

This update is blocked by yet another issue. `reductive` switched to `rand` 0.7. However, `rand` 0.7 makes use of crate renaming, which seems to be used seldomly enough that it is not supported by Nix' `buildRustCrate`. Since I use Nix to build Docker images for sticker (which uses finalfusion), I would like to see that fixed before making the leap. I have already submitted a PR in nixpkgs to add support for crate renames.

**Comment by danieldk** [comment]
Fixed by #54.
